api-controllers/ServiceFabrikApiController.js

```diff
@@ -347,7 +347,7 @@ class ServiceFabrikApiController extends FabrikBaseController {
           restore_guid: metadata.restore_guid,
           instance_guid: req.params.instance_id,
           arguments: _.assign({
-            backup: _.pick(metadata, 'type', 'secret')
+            backup: _.pick(metadata, 'type', 'secret', 'snapshotId', 'started_at', 'finished_at')
```

Comment:
@jayeshjk : Can you take care of this?

Reply:
Oh, I had missed this. Will push the change.

Reply:
On a closer look, `snapshotId` is being used as field name in backup operation itself, see [here](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/operators/backup-operator/BackupService.js#L229), which implies that field name `snapshotId` is already in use in BackupFiles and Backup Resources. To keep the consistent naming convention, changes must be started from this point. On a first look, changes within SF codebase seem feasible, but for just sanity check should we also confirm whether services teams are making use of our BackupFiles internally? Also if we can live with it for now, this can be taken up in a separate PR to avoid incorporating changes in unrelated files in this PR.  @sauravmndl @abh1kg kindly share your thoughts.

Reply:
Will be addressed in future PRs as stated above. Hence keeping unresolved.